charts/merlin/templates/mlobs-bootstrap-job.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.mlobs.bootstrap.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "merlin.resource-prefix" . }}-mlobs-bootstrap
+  annotations:
+    "helm.sh/hook": post-upgrade,post-install
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  template:
+    spec:
```

Comment:
Would be good to have option to add labels on podtemplate spec if there are labels present